src/scripts/browserdeviceprofile.js

```diff
@@ -750,8 +773,26 @@ define(['browser'], function (browser) {
         var maxH264Level = browser.chromecast ? 42 : 51;
         var h264Profiles = 'high|main|baseline|constrained baseline';
 
-        if (maxH264Level >= 51 && browser.chrome && !browser.osx) {
-            h264Profiles += '|high 10';
+        if (browser.tizen || browser.orsay || browser.web0s ||
+            videoTestElement.canPlayType('video/mp4; codecs="avc1.640833"').replace(/no/, '')) {
+            maxH264Level = 51;
```

Comment:
it is already `51` if it's not a chromecast, and if it is this test won't probably pass... something strange is going on here

Reply:
I will let `maxH264Level = 42` at first.
I passed the test `canPlayType('video/mp4; codecs="avc1.640833"')` on `Chrome` `Firefox` `Safari` `Edge` `new Edge`.
This [doc](https://developers.google.com/cast/docs/media) says that only Chromecast Ultra can play up to level 5.2 but I dont have an Ultra
 now.